package.json

```diff
@@ -66,5 +67,9 @@
     "karma-chai": "^0.1.0",
     "karma-chrome-launcher": "^2.2.0",
     "karma-mocha": "^1.3.0"
+  },
+  "nodemonConfig": {
+    "watch": ["dist/node/*"],
+    "delay": "2500"
```

Comment:
Adding a delay to avoid constantly restarting the Node server during bootup, or multiple times during a build. We can tweak this value if we find it's too long or too short.